go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/mattermost/mattermost-plugin-boards
 
-go 1.22.0
-
-toolchain go1.22.8
+go 1.24.0
```

Comment:
nit: since we're updating it, let's use the following to get the latest toolchain version:

```suggestion
go 1.24.0

toolchain go1.24.5
```

Reply:
@esarafianou AFAIK toolchain definition is optional in 1.24 will pick the latest. Is there any reason security-wise that you want us to set it to a specific version?

Reply:
I wasn't aware of this change in 1.24. Feel free to ignore my suggestion.